lib/internal/crypto/webcrypto.js

```diff
@@ -690,6 +692,13 @@ const subtle = new SubtleCrypto();
 class Crypto {}
 const crypto = new Crypto();
 
+function getRandomValues(_) {
```

Comment:
nit: For better readability (I don't know if the linter will complain)
```suggestion
function getRandomValues(data) {
```

Reply:
The spec calls it `array`: https://w3c.github.io/webcrypto/#crypto-interface
```suggestion
function getRandomValues(array) {
```